qemu/tests/cfg/qemu_disk_img_create.cfg

```diff
@@ -0,0 +1,51 @@
+- qemu_disk_img_create:
+    no qed raw vmdk
+    type = create_external_snapshot_via_driver
+    kill_vm = yes
+    start_vm = no
+    md5sum_bin = "md5sum"
+    force_create_image = no
+    backup_image_before_testing = yes
+    restore_image_before_testing = yes
+    tmp_dir = /var/tmp
+    image_chain = "image1 sn1"
+    image_name_sn1 = "images/sn1"
+    image_format_sn1 = "qcow2"
+    tmp_file_name = ${tmp_dir}/test.img
+    guest_temp_file_sn1 = "${tmp_dir}/sn1"
+    guest_temp_file_image1 = "${tmp_dir}/sn1"
+    backing_chain = yes
+    remote_host = localhost
+    file_create_cmd = "dd if=/dev/urandom of=%s bs=4k count=250"
+
+    variants:
+        - protocol_ssh:
+            subcommand = create_ssh
+            check_image = no
+            post_check = no
+            external_snapshot_driver = ssh
+            file_host = localhost
+            file_host_key_check = no
+            ssh_key_dir = /root/.ssh
+            create_ssh_key_cmd = "ssh-keygen -t rsa -q -N '' -f /root/.ssh/id_rsa"
+            copy_pub_key_cmd = "cat ${ssh_key_dir}/id_rsa.pub > ${ssh_key_dir}/authorized_keys"
+            user_profile = "/root/.bash_profile"
+            user_profile_backup = "/root/bk.bash_profile"
+            boot_ssh_agent = "ssh-agent >> ${user_profile};source ${user_profile};ssh-add;"
```

Comment:
I think we just need to record SSH_AUTH_SOCK and SSH_AGENT_PID. And export them before executing qemu-img and qemu-kvm. If you use profile, we also need to remove it after test.